leaflet/leaflet.d.ts

```diff
@@ -1231,8 +1231,7 @@ declare namespace L {
         panTo(latlng: LatLngTuple, options?: PanOptions): this;
         panBy(offset: Point): this;
         panBy(offset: PointTuple): this;
-        setMaxBounds(bounds: Bounds): this; // is this really bounds and not lanlngbounds?
-        setMaxBounds(bounds: BoundsLiteral): this;
+        setMaxBounds(bounds: LatLngBoundsExpression): this; // is this really bounds and not lanlngbounds?
```

Comment:
The documentation incorrectly(?) lists the parameter type as `Bounds`. If you are sure that it is `LatLngBounds`, you can remove the comment.
Can you also split it into two declarations: one for `LatLngBounds` and another for `LatLngBoundsLiteral` so that we can see overloads.
Thanks in advance.